content/blog/2023/2023-08-25-fairness-reweighing-preprocessor.md

```diff
@@ -0,0 +1,51 @@
++++
+author = "Žan Mervič"
+date = "2023-08-25"
+draft = false
+title = "Orange Fairness - Reweighing as a preprocessor"
+type = "blog"
+thumbImage = "/blog_img/2023/2023-08-25-fairness-reweighing-preprocessor-thumb.png"
+frontPageImage = "/blog_img/2023/2023-08-25-fairness-reweighing-preprocessor-thumb.png"
+blog = ["fairness", "reweighing"]
+shortExcerpt = "Expanding on the Orange fairness Reweighing widget: using it as a preprocessor and integrating new fairness scoring metrics."
+longExcerpt = "Diving deeper into the Orange fairness Reweighing widget, we explore its use as a preprocessor for models. Discover the new widgets and fairness scoring metrics; all illustrated using the German credit dataset, supplemented with visual insights through box plots."
++++
+
+In the [previous blog post]({{< ref "/blog/2023/2023-08-24-fairness-reweighing-dataset.md" >}}), we introduced the Orange fairness Reweighing widget and used it to reweigh a dataset. In this blog, we will explore another use case for the Reweighing widget: using it as a preprocessor for a specific model. Doing so will allow us to use the reweighing algorithm when evaluating model performance using cross-validation, random sampling, etc.
+
+### Weighted Logistic Regression and Combine Preprocessors:
```

Comment:
This paragraph is too technical for the current (top) position. This should be somewhere, but more of a side-note when you describe the workflow. Especially since these are probably very temporary.